frontend/src/components/Sidebar/RequirementSection.tsx

```diff
@@ -121,17 +123,21 @@ export class RequirementSection extends React.Component<
     }
 
     return (
-      <div>
+      <div key={index.toString()}>
         <ANDORText>{this.convertTypeToText(req.type)}</ANDORText>
         {req.courses
           .filter(c => c.type === "COURSE")
           .map(c => this.renderCourse(c as IRequiredCourse))}
         {req.courses
           .filter(c => c.type === "AND")
-          .map(c => this.renderRequirement(c))}
+          .map((c: Requirement, index: number) =>
+            this.renderRequirement(c, index)
+          )}
         {req.courses
           .filter(c => c.type === "OR")
-          .map(c => this.renderRequirement(c))}
+          .map((c: Requirement, index: number) =>
+            this.renderRequirement(c, index)
+          )}
```

Comment:
This should probably go in a separate PR, but could you change the requirements to be displayed using prefix + parens? Ex. (OR (AND CS2500 CS2501) EECE2160). The parens really clear stuff up, rather than having to look carefully at indentation, for example with the science requirement of CS.

Reply:
Made a card for this in Trello